scripts/combine/setupCombineWMass.py

```diff
@@ -310,9 +311,20 @@
 )
 
 if not args.wlike:
-    cardTool.addLnNSystematic("CMS_Fakes", processes=[args.qcdProcessName], size=1.05)
+    cardTool.addLnNSystematic("CMS_Fakes", processes=[args.qcdProcessName], size=1.05, group="MultijetBkg")
     cardTool.addLnNSystematic("CMS_Top", processes=["Top"], size=1.06)
     cardTool.addLnNSystematic("CMS_VV", processes=["Diboson"], size=1.16)
+
+    # FIXME: it doesn't really make sense to mirror this one since the systematic goes only in one direction
```

Comment:
So what do you propose to do?

Reply:
Let's leave it like this for now.
 
I am cooking up something to make an actual envelope changing the pt threshold above and below the nominal value, but I actually think we shouldn't try to decrease the threshold below 30 GeV. 
One solution is either to increase the nominal from 30 to 35 or even 40 (but I don't want to remove too much stats from the fake region, even though it should increase the QCD/EWK ratio), or just to define the opposite histogram variation as equal to the nominal, so to have a one-sided band